contracts/controller/src/views.rs

```diff
@@ -82,26 +82,27 @@ impl Contract {
 
         let potential_borrow =
             (supplies.0 * RATIO_DECIMALS.0 / self.health_threshold.0) - gotten_borrow.0;
-        let price = self.get_price(dtoken_id).unwrap().value.0;
+        let price = Balance::from(self.get_price(dtoken_id).unwrap().value);
```

Comment:
@dmytro-shash price is not a `Balance` (that is actually type of mess we trying to avoid introducing incompatible types)

let's rewrite this in the following way
```
let price = self.get_price(dtoken_id).unwrap();
WBalance::from(potential_borrow * ONE_TOKEN / price.value.0)
```

@pahomas why do we here multiply by price.value but not divide by price.fraction_digits ?